tests/tests/operation_tests.cpp

```diff
@@ -4118,6 +4118,173 @@ BOOST_FIXTURE_TEST_SUITE(operation_tests, clean_database_fixture)
+
+            tx.clear();
+
+            BOOST_TEST_MESSAGE("--- Creating account chewy with alice");
+            acc_create.fee = ASSET("0.000 GOLOS");
+            acc_create.creator = "alice";
+            acc_create.new_account_name = "chewy";
+            acc_create.owner = authority(1, generate_private_key("chewy_owner").get_public_key(), 1);
+            acc_create.active = authority(1, generate_private_key("chewy_active").get_public_key(), 1);
+            acc_create.posting = authority(1, generate_private_key("chewy_posting").get_public_key(), 1);
+            acc_create.memo_key = generate_private_key("chewy_memo").get_public_key();
+            acc_create.json_metadata = "";
+
+            tx.operations.push_back(acc_create);
+
+            BOOST_TEST_MESSAGE("--- Creating account donna with alice");
+            acc_create.fee = ASSET("0.000 GOLOS");
+            acc_create.creator = "alice";
+            acc_create.new_account_name = "donna";
+            acc_create.owner = authority(1, generate_private_key("donna_owner").get_public_key(), 1);
+            acc_create.active = authority(1, generate_private_key("donna_active").get_public_key(), 1);
+            acc_create.posting = authority(1, generate_private_key("donna_posting").get_public_key(), 1);
+            acc_create.memo_key = generate_private_key("donna_memo").get_public_key();
+            acc_create.json_metadata = "";
+
+            tx.operations.push_back(acc_create);
+            tx.sign(alice_private_key, db->get_chain_id());
+
+            BOOST_CHECK_NO_THROW(db->push_transaction(tx, 0));
+
+            BOOST_TEST_MESSAGE("--- Set vesting withdraw routes");
+
+            set_withdraw_vesting_route_operation to_chewy;
+            to_chewy.from_account = "bob";
+            to_chewy.to_account = "chewy";
+            to_chewy.percent = 5000;
+
+            set_withdraw_vesting_route_operation to_donna;
+
+            to_donna.from_account = "bob";
+            to_donna.to_account = "donna";
+            to_donna.percent = 5000;
+
+            tx.clear();
+
+            BOOST_CHECK_NO_THROW(push_tx_with_ops(tx, bob_private_key, to_chewy, to_donna));
+
+            BOOST_TEST_MESSAGE("--- Start vesting withdraw");
+
+
+            withdraw_vesting_operation withdraw_vesting_op;
+            withdraw_vesting_op.account = "bob";
+            withdraw_vesting_op.vesting_shares = asset(db->get_account("bob").vesting_shares.amount / 2, VESTS_SYMBOL);
+
+            tx.clear();
+            tx.operations.push_back(withdraw_vesting_op);
+
+            tx.set_expiration(db->head_block_time() + STEEMIT_MAX_TIME_UNTIL_EXPIRATION);
+
+            tx.sign(bob_private_key, db->get_chain_id());
+
+            BOOST_CHECK_NO_THROW(db->push_transaction(tx, 0));
+
+            BOOST_TEST_MESSAGE("--- Generating block up to first withdrawal");
+
+            generate_blocks(db->head_block_time() + STEEMIT_VESTING_WITHDRAW_INTERVAL_SECONDS, true);
+
+            {
```

Comment:
Do not use { } blocks without conditions, loops and etc. If you want separate code block from another - use comment or BOOST_TEST_MESSAGE.
Also same in another places.

Reply:
Why? I want to create a local variable which has to exist in this local space. It is not needed somewhere else. { } is used for preventing creation variables like 
const var& a1 = ....
.....
const var& a2 = ....

I think
{
const var& a =...
}

{
const var&a = ...
} 

looks better.

Reply:
@e-schepachev 
1. Ok. But what if we modifying this code in future and variable is need to be outside block? We should unindent block, remove brackets - it is our work time and energy.

2. Indents and nesting are always complicating readability.

3. And mainly. Just remember:
Your project - your style.
Foreign project - foreign style.
Disputes are profitless in most cases.
:)

Reply:
1. Any code modification takes our time and energy. We may need some more variables the same type and finding the vacant name will also take our time if we remove braces.
2. We have no functions where we could move this logic part in this test suite. So intending it is a rather small payment for  keeping  namespace clear
3. This style is used in this project. It's not my invention.

Reply:
@e-schepachev 
1. If we need second same variable - it is better to reuse this variable.
If we need second another variable - we know how it is another and naming isn't a problem.
3. Where is it used? Project is a fork of STEEM, our code is good, but STEEM's isn't. If it was STEEM's code, it couldn't be used as example.

Reply:
1. No it is not. Sometimes we could not reuse variable. For example constant references. In such cases we have to create new one. You have a big test which contains 15 variables user_xx. And 17 variables another_user_yy. Addition one more variable of this type with the same semantic will be a problem - you have to find the last used name.  
2. May be it is not a bad idea?

Reply:
Also sometimes easier to create the new variable than clear old.

Reply:
I try to show you that using the { } to reduce variable visibility scope makes code more clear than creating variables like var_xx. More clear is more readable.

> We do not clearing anything. It is not a C with char* and etc. We just setting new value to old variable with =.

I mean that. Sometimes construct new object is easier than set all its fields  "default" value.

Reply:
@e-schepachev  We could think about { } blocks in future but this issue should be done without them.

Reply:
Why? 
I need two variables bob. I don't want to have code like:

const auto& bob_1 = ...
...

const auto& bob_2 = ... 
const auto& alice_1 = ...
const auto& donna_1 = ...

It looks like a typo

Reply:
@e-schepachev Ok, use { } blocks.